pragma/tests/fetchers_test.py

```diff
@@ -317,6 +126,29 @@ def future_fetcher_config(request):
     return request.param
 
 
+@pytest.fixture
+def other_mock_endpoints(future_fetcher_config):
+    # fetchers such as OkxFutureFetcher and BinanceFutureFetcher
+    # have other API endpoints that must be mocked
+    fetcher = future_fetcher_config["fetcher_class"](
+        SAMPLE_FUTURE_ASSETS, PUBLISHER_NAME
+    )
+    other_mock_fns = future_fetcher_config.get("other_mock_fns", {})
+    if not other_mock_fns:
+        return []
+
+    responses = []
+    for asset in ("BTC", "ETH"):
+        for mock_fn in other_mock_fns:
+            [*fn], [*val] = zip(*mock_fn.items())
+            fn, val = fn[0], val[0]
+            url = getattr(fetcher, fn)(**val["kwargs"][asset])
+            with open(val["mock_file"], "r") as f:
+                mock_file = json.load(f)
+            responses.append({"url": url, "json": mock_file[asset]})
+    return responses
```

Comment:
@EvolveArt here's the fixturized `other_mock_endpoints`. It's returning a list of dictionaries with urls and json responses so that it can be registered with the requests/aioresponses mock in the tests themselves.

Reply:
Looks good! 

Could we fetch these dynamically tho : `for asset in ("BTC", "ETH"):` ?

Reply:
Done! 🙂